test/setOption.html

```diff
@@ -155,5 +156,44 @@
             });
 
         </script>
+
+        <script>
+            require([
+                'echarts'/*, 'map/js/china' */
+            ], function (echarts) {
+                var option;
+                option = {
+                    xAxis: {
+                        type: 'category',
+                        boundaryGap: false,
+                        data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
+                    },
+                    yAxis: {
+                        type: 'value'
+                    },
+                    series: [
+                        {
+                        data: [820, 932, 901, 934, 1290, 1330, 1320],
+                        type: 'line',
+                        areaStyle: {}
+                        }
+                    ]
+                };
+                chart = myChart = testHelper.create(echarts, 'main1', {
+                    title: 'Click the two buttons. Animation should be appropriate, and dataZoom state should be kept.',
```

Comment:
The title should be related to the current test case. Here it can be like
**_The area polygon should be removed after the button is clicked_**.